README.md

```diff
@@ -4,6 +4,20 @@ This bot is used to generate and maintain text channels for courses taken by uni
 
 Commands are restricted to discord users with roles authorised in the config file under "trustedRoles"
 
+## Setup
```

Comment:
This is great thankyou, I've been meaning to do this for a hot minute
Could you include a link to here to get OAuth token pls
https://discord.com/developers/applications